data/json/furniture_and_terrain/terrain-windows.json

```diff
@@ -3918,24 +3918,22 @@
     "looks_like": "t_window_empty",
     "symbol": "0",
     "color": "light_cyan",
-    "move_cost": 0,
+    "move_cost": 2,
```

Comment:
I'm not sure that skylight frame should be as easy to walk upon as a pavement.

Reply:
For some reason, it doesn't allow me to climb down through it when the move_cost is 0.

Reply:
Well the `"trap": "tr_ledge"` Doesn't trigger when move_cost is 0.